app/src/main/java/com/swent/assos/model/navigation/NavigationGraph.kt

```diff
@@ -8,21 +8,25 @@ import androidx.navigation.compose.rememberNavController
 import com.swent.assos.model.view.AppViewModel
 import com.swent.assos.ui.login.LoginScreen
 import com.swent.assos.ui.login.SignUpScreen
-import com.swent.assos.ui.screens.Settings
 import com.swent.assos.ui.screens.assoDetails.AssoDetails
 import com.swent.assos.ui.screens.assoDetails.EventDetails
 import com.swent.assos.ui.screens.assoDetails.NewsDetails
 import com.swent.assos.ui.screens.manageAsso.CreateEvent
 import com.swent.assos.ui.screens.manageAsso.CreateNews
 import com.swent.assos.ui.screens.manageAsso.ManageAssociation
+import com.swent.assos.ui.screens.profile.Appearance
+import com.swent.assos.ui.screens.profile.Following
+import com.swent.assos.ui.screens.profile.MyAssociations
+import com.swent.assos.ui.screens.profile.NotificationSettings
+import com.swent.assos.ui.screens.profile.Settings
 
 @Composable
 fun NavigationGraph() {
   val navController = rememberNavController()
   val navigationActions = NavigationActions(navController = navController)
   val appViewModel: AppViewModel = hiltViewModel()
 
-  NavHost(navController = navController, startDestination = Destinations.LOGIN.route) {
+  NavHost(navController = navController, startDestination = Destinations.HOME.route) {
```

Comment:
You forgot to change this line on the push 
```suggestion
  NavHost(navController = navController, startDestination = Destinations.LOGIN.route) {
```